CrowServe/Cbor.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "common.h"
+
+template<typename T>
+bool deserialize_cbor_into((T& container, const std::span<const char>& input)) {
+    static_assert(std::is_trivial_v<T>);
+    // TODO: error handling
+    Json j = Json::from_cbor(input);
```

Comment:
Not important, I tend to name them json, that's why I alias nlohmann::json as Json.